Automattic/jetpack
# VideoPress: Fix minimum starting point for preview on hover feature with very short videos

Fixes #29965

## Proposed changes:

* Checks if the video is shorter than the minimum loop duration, disabling the controls and fixing the values in that case

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor
* Add a VideoPress video block with a very short video (< 3s) *
* Enable the preview on hover feature
* Check that the starting point is set to 0
* Check that the max loop value is set to the video duration
* Check that the state is set correctly (`attributes.posterData` in the block)
* Check that videos longer than 3 seconds are not affected and the feature keeps working as usual for these cases

![2023-04-07_16-17-57](https://user-images.githubusercontent.com/8486249/230665001-971b7c5c-357e-4bcc-aef7-1b89f6288ab3.png)

* You can cut a video to less than 3 seconds [with ffmpeg](https://stackoverflow.com/questions/18444194/cutting-the-videos-based-on-start-and-end-time-using-ffmpeg) or using an online tool like [this one](https://clideo.com/editor/cut-video).
